src/bunfig.zig

```diff
@@ -207,6 +207,11 @@ pub const Bunfig = struct {
                 if (json.get("preload")) |expr| {
                     try this.loadPreload(allocator, expr);
                 }
+
+                if (json.get("disableTelemetry")) |expr| {
```

Comment:
Instead of inverting the logic, we can just do `telemetry = false`

Reply:
Thank you for your feedback, I will fix this :)